docs/reference/configuration/keywords.rst

```diff
@@ -1503,9 +1503,7 @@ but required when installing ERT at a new site.
 .. _schedule_prediction_file:
 .. topic:: SCHEDULE_PREDICTION_FILE
 
-        This is the name of a schedule prediction file. It can contain %d to get
-        different files for different members. Observe that the ECLIPSE datafile
-        should include only one schedule file, even if you are doing predictions.
```

Comment:
I would keep the documentation until we actually remove the keyword, but say "This keyword is deprecated and will be removed." at the top.

Reply:
very well! it is done.